vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -15060,7 +15110,7 @@ unit:OZ_M-FT
   qudt:iec61360Code "0112/2///62720#UAB133" ;
   qudt:uneceCommonCode "4R" ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
-  rdfs:label "OZ_M_US FT" ;
+  rdfs:label "OZ_M FT" ;
```

Comment:
Something wrong here. The unit is OHM_Ab, and the label is OZ_M.

Reply:
Steve,

I am only seeing OHM_Ab for OHM_Ab, and I only see OZ_M FT for OZ_M-FT